config/database.yml

```diff
@@ -1,16 +1,18 @@
-development:
+default: &default
   adapter: postgresql
-  database: growstuff_dev
-  host: localhost
+  encoding: unicode
+  host: db
+  port: 5432
   user: postgres
-  password: postgres
+  password: <%= ENV['POSTGRES_PASSWORD'] %>
```

Comment:
I don't know if it's possible to use our existing `environment.yaml` - docker-compose expects environment files to be in `NAME=VALUE` format.